ui/src/adapters/api/credentials.ts

```diff
@@ -15,6 +14,7 @@ import {
 } from "src/adapters/api";
 import { getStrictParser } from "src/adapters/parsers";
 import { Credential, Env, Json, Link, LinkStatus } from "src/domain";
+import { ProofTypes } from "src/domain/credential";
```

Comment:
Wrong import

Reply:
If we add the exports to the `index.ts` the auto import works as expected. Having everything exported from `index.ts` was an experiment that we can decide to remove at any point, but it has 2 benefits:
 * Make sure we have no collisions in interface names, which can lead to weird bugs since TS would merge them.
 * Have a single import statement from the domain per module.